README.md

```diff
@@ -46,33 +48,16 @@ cd stylised-controllable-image-captioning-StreamlitApp
 
 Build and run the docker image locally:
 ```
-docker build -t ctrl-img-cap-streamlit:latest .
-docker run -d -p 8501:8501 ctrl-img-cap-streamlit:latest
+make run
```

Comment:
will this build work for Windows? is it easy to extend the support of docker build to Windows?
if not, better to mention which OS works for this docker build?

Reply:
Unfortunately, my experience with Windows is very limited -- so I don't know the answer to either of those questions.

AFAIK, there is a way to install UNIX tool equivalents like `make` and `wget` in Windows but I don't know for sure.

I suggest that we create an issue to add Windows support and leave this to someone familiar with software development on Windows.